components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/Main.java

```diff
@@ -54,7 +54,7 @@ public static void main(String... args) {
             logger.error(MICRO_GW + ": Error occurred while executing command.", e);
             Runtime.getRuntime().exit(1);
         } catch (CLIInternalException e) {
-            outStream.println(MICRO_GW + ":" + " + INTERNAL_ERROR_MESSAGE" + " - " + e.getMessage());
+            outStream.println(MICRO_GW + ":" + INTERNAL_ERROR_MESSAGE + " - " + e.getMessage());
```

Comment:
W can avoid creating a string for `:`
```suggestion
            outStream.println(MICRO_GW + ':' + INTERNAL_ERROR_MESSAGE + " - " + e.getMessage());
```